src/odemis/acq/align/roi_autofocus.py

```diff
@@ -119,22 +126,29 @@ def autofocus_in_roi(
         ccd: model.Component,
         focus: model.Component,
         focus_range: tuple,
-        nx: int = 3,
-        ny: int = 3,
+        n_focus_points: tuple = (3, 3),
```

Comment:
Modify the test cases for this function once the earlier pull requests with test cases gets merged